java/monkey/src/main/java/Lexer.java

```diff
@@ -59,10 +82,14 @@ private void advance() {
     }
 
     private char getCc() {
-        if (this.pos >= this.input.length() || this.pos < 0) {
+        return this.peek(this.pos);
+    }
+
+    private char peek(int pos) {
```

Comment:
Can we add in this function?
```java
private char peekNext() {
    return peek(this.pos + 1);
}
```

Reply:
Of course! It makes things clearer. Added in my last commit.